mysql/tests/common.py

```diff
@@ -21,3 +21,16 @@
 
 USER = 'dog'
 PASS = 'dog'
+
+
+class MockCursor:
+    def execute(self, command):
+        raise UnicodeDecodeError('encoding', b'object', 0, 1, command)
+
+    def close(self):
+        return MockCursor()
+
+
+class MockDatabase:
+    def cursor(self):
+        return MockCursor()
```

Comment:
These seem specific to the `test_innodb_status_unicode_error` test, should we move them to inside the test function body instead? Doesn't seem ideal to have them in `common.py` with generic names like `MockCursor`.